src/System.Text.Primitives/System/Text/Parsing/InvariantUri.tt

```diff
@@ -0,0 +1,113 @@
+﻿<#@ template hostspecific="false" language="C#" #>
```

Comment:
This needs some security focused review before we merge. Feel free to remove this one parser from this PR so we can merge sooner.